azure-spring-boot-bom/pom.xml

```diff
@@ -118,6 +119,11 @@
                 <artifactId>azure-spring-boot</artifactId>
                 <version>${azure.spring.boot.version}</version>
             </dependency>
+            <dependency>
```

Comment:
No need to include this in a dependencyManagement here since we already got this covered in the jms starter